data/data/bootstrap/baremetal/files/etc/containers/systemd/ironic.container.template

```diff
@@ -18,6 +18,10 @@ Exec=/bin/runironic
 Network=host
 Volume=${AUTH_DIR}:/auth:z,ro
 Volume=ironic.volume:/shared:z
+{{ if .PlatformData.BareMetal.DisableIronicVirtualMediaTLS }}
+Volume=ironic-virtualmedia-tls-cert.yaml:/certs/vmedia/tls.crt:z
```

Comment:
I dont know which directory the ironic-virtualmedia-tls-cert.yaml file will be available in, and same for the key file.

Reply:
It seems that if you move the template to `data/data/bootstrap/baremetal/files/etc/ironic/`, it will be available in `/etc/ironic` on the bootstrap.